### PR TITLE
Add coq-io-hello-world.1.2.0

### PR DESCRIPTION
**Pull request by clarus**
Review please :pray:

### CODE REVIEW

File: released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+  [make "-C" "extraction"]
+]
+install: [
+  ["install" "-T" "extraction/main.native" "%{bin}%/helloWorld"]
+]
+depends: [
+  "coq-io" {>= "4.0.0" build}
```
Comment (palmskog) [inline released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam]:
```suggestion
  "coq-io" {>= "4.0.0" & build}
```

File: released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "coq-io-system" {build}
+  "coq-io-system-ocaml" {>= "2.3.0"}
+]
+tags: [
+  "keyword:effects"
+  "keyword:extraction"
```
Comment (palmskog) [inline released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam]: Please add a `"date:YYYY-MM-DD"` tag (indicating when you did the release on GitHub).

File: released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+license: "MIT"
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+  [make "-C" "extraction"]
+]
+install: [
+  ["install" "-T" "extraction/main.native" "%{bin}%/helloWorld"]
+]
+depends: [
```
Comment (palmskog) [inline released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam]: Just to be clear, this package doesn't require Coq itself in any way (other than perhaps implicitly via dependencies)?

Comment (clarus) [reply released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam]: Yes it does depend on Coq actually, so I added that explicitly.

**Pull request merged**